apps/meteor/client/views/admin/import/NewImportPage.tsx

```diff
@@ -86,6 +86,20 @@ function NewImportPage() {
 
 	const [files, setFiles] = useState<File[]>([]);
 
+	// CSV importer expects ZIP files, Slack Users expects CSV files, most other importers expect ZIP files
+	const acceptedFileTypes = useMemo(() => {
+    if (importer?.key === 'csv') {
+        return '.csv,.zip,text/csv,application/zip,application/x-zip,application/x-zip-compressed';
+    }
+    if (importer?.key === 'slack-users') {
+        return '.csv,text/csv';
+    }
+    if (importer) {
```

Comment:
Also make it default to unfiltered instead of zip if the importer doesn't specify its type.